howdoi/howdoi.py

```diff
@@ -581,12 +582,15 @@ def howdoi(raw_query):
     res = cache.get(cache_key)  # pylint: disable=assignment-from-none
 
     if res:
+        logging.info('Using cached response (add -C to clear the cache)')
         return _parse_cmd(args, res)
 
+    print(f'Fetching answers for query: {args["query"]}')
+
     try:
         res = _get_answers(args)
-        if not res:
-            res = {'error': 'Sorry, couldn\'t find any help with that topic\n'}
+        if not res and not args["explain"]:  # only set message when they havent set the -x flag
```

Comment:
Here, we always want to return `Sorry, couldn't find any help with that topic`

However, if explain was not used, we will want to append to this string `(use --explain to learn why)`